src/lib/svg_text_utils.js

```diff
@@ -91,7 +91,7 @@ exports.convertToTspans = function(_context, _callback) {
         }
         _context.text('')
             .style({
-                visibility: 'visible',
+                visibility: 'inherit',
```

Comment:
Good to know about. That's the sort of obscure possibility for interaction that I thought maybe could exist.